docs/schemas/FeatureManagement.v1.0.0.json

```diff
@@ -0,0 +1,75 @@
+{ 
+    "$schema":"http://json-schema.org/draft-07/schema#", 
+    "$id":"http://azconfig.io/schemas/FeatureManagement-DotNet/v1.0.0/FeatureManagement.json", 
+    "title":"Decalaration of features in the Microsoft.FeatureManagement library.", 
+    "definitions": { 
+        "FeatureFlag": { 
+            "type":"object", 
+            "properties": { 
+                "EnabledFor": { 
+                    "type":"array", 
+                    "items": { 
+                        "type":"object", 
+                        "required":[ 
+                            "Name" 
```

Comment:
We also support this format for OFF features:
```
 "FeatureU": {
    "EnabledFor": []
}
```
Should we remove "Name" as a required property here?

Reply:
Scratch that. I misunderstood. Name is a required property for the objects inside EnabledFor array.
It's required because we don't support something like ` "EnabledFor": [{}]`